memory/numa_test.py

```diff
@@ -43,8 +43,10 @@ def setUp(self):
             'nr_pages', default=memsize / memory.get_page_size())
         self.map_type = self.params.get('map_type', default='private')
 
-        if len(memory.numa_nodes()) < 2:
-            self.cancel('Test requires two numa nodes to run')
+        nodes = memory.numa_nodes_with_memory()
+        if len(nodes) < 2:
```

Comment:
`len(memory.numa_nodes_with_memory())`

Reply:
It was used as if the condition is met, same is displayed in message at https://github.com/avocado-framework-tests/avocado-misc-tests/pull/908/files#diff-082c2aa8ea45734f25c0147833e7a73bR49.

Reply:
Ok, got it. Missed the usage in the next line